### PR TITLE
Turku theme fixes

### PR DESCRIPTION
**Pull request by juhakujala**
* Update styles and style variables
* Change turku logo files to correct ones
* Use turku primary color on the default-image.svg
* Add new turku specific translation strings
* Remove unecessary images

localhost_8086_thfu2

### CODE REVIEW

File: assets/sass/app.scss [hunk 1]
```diff
@@ -3,3 +3,6 @@
 @import "~kerrokantasi-ui/assets/sass/kerrokantasi/variables";
 @import "~kerrokantasi-ui/assets/sass/kerrokantasi/bootstrap";
 @import "~kerrokantasi-ui/assets/sass/kerrokantasi/kerrokantasi";
+
+@import "turku/navigation";
```
Comment (frwickst) [inline assets/sass/app.scss:7]: Does this work even if the file is called _navigation.scss?

File: assets/sass/app.scss [hunk 2]
```diff
@@ -3,3 +3,6 @@
...
+@import "turku/labels";
```
Comment (frwickst) [inline assets/sass/app.scss:8]: Does this work even if the file is called _labels.scss?

**Review by frwickst — CHANGES_REQUESTED** [review]
Overall everything looks good, but the imports look a bit strange to me. Even if it for some reason works, I see no reason to have the `_` in there, or is there a good reason for it?

**Comment by juhakujala** [comment]
> Overall everything looks good, but the imports look a bit strange to me. Even if it for some reason works, I see no reason to have the `_` in there, or is there a good reason for it?

They do work. It's a convention that is also used in the kerrokantasi-ui repository now. The `_` implies that the file is sort of a component or a minor part of the styles. Same convention can be found from the bootstrap repository. https://github.com/twbs/bootstrap/tree/master/scss

**Review by frwickst — APPROVED** [review]

**Pull request merged**